applications/GeoMechanicsApplication/python_scripts/geomechanics_T_solver.py

```diff
@@ -0,0 +1,170 @@
+import KratosMultiphysics
+import KratosMultiphysics.GeoMechanicsApplication as KratosGeo
+from KratosMultiphysics.GeoMechanicsApplication.geomechanics_solver import GeoMechanicalSolver as GeoSolver
+
+
+def CreateSolver(model, custom_settings):
+    return TSolver(model, custom_settings)
+
+
+class TSolver(GeoSolver):
+    """Solver for the solution of thermal problems."""
+
+    def __init__(self, model, custom_settings):
+        super().__init__(model, custom_settings)
+
+        KratosMultiphysics.Logger.PrintInfo("GeoMechanics_T_Solver", "Construction of Solver finished.")
+
+    @classmethod
+    def GetDefaultParameters(cls):
+        this_defaults = KratosMultiphysics.Parameters("""{
+            "solver_type": "geomechanics_T_solver",
+            "model_part_name": "PorousDomain",
+            "domain_size": 2,
+            "start_time": 0.0,
```

Comment:
If I'm not mistaken, for the other solvers `"start_time"` and `"end_time"` are no longer stored under `"solver_settings"`, but under `"problem_data"`. Perhaps we can follow the same scheme here?

Reply:
Thank you for pointing. I just removed "start_time"and "end_time" from defaults.